app/js/components/License.jsx

```diff
@@ -26,7 +26,9 @@ const License = (props) => (
     <CardBody>
       {props.error && <Alert color="danger">{props.error}</Alert>}
       {props.isLicenseOwner ? <IsLicenseOwner t={props.t}/> : <BuyLicense buyLicense={props.buyLicense} t={props.t}/>}
-      <p>{props.t('license.rating')} {props.userRating ? props.userRating : '-'}</p>
+      {props.isLicenseOwner &&
+        <p>{props.t('license.rating')} {props.userRating ? props.userRating : 'no rating available yet'}</p>
```

Comment:
```suggestion
        <p>{props.t('license.rating')} {props.userRating ? props.userRating : t('license.noRating')}</p>
```
Also, please capitalize the sentence